modules/luci-base/htdocs/luci-static/resources/network.js

```diff
@@ -3374,7 +3374,7 @@ WifiDevice = baseclass.extend(/** @lends LuCI.network.WifiDevice.prototype */ {
 	 */
 	getHWModes: function() {
 		var hwmodes = this.ubus('dev', 'iwinfo', 'hwmodes');
-		return Array.isArray(hwmodes) ? hwmodes : [ 'b', 'g' ];
+		return Array.isArray(hwmodes) ? hwmodes : [ 'b', 'g', 'a', 'ac', 'ax', 'be' ];
```

Comment:
@jow- probably knows better, but I am not sure if adding "be" capabilities would cause all these to be added (as they have not been there so far).

Reply:
I was wondering whether `a` should be there. It's the older a standard, right?

Reply:
if I do a ` ubus call luci-rpc getWirelessDevices` I get hwmodes_text like this (output from radio1 on the banana rpi-4)

Thought to align this code to iwinfo / rpcd

![image](https://github.com/user-attachments/assets/989dcf39-915f-48f5-95b6-12fc694c8273)

Reply:
Then at least `a` -> `n` change?

Reply:
yes sure will replace with 'n'